src/diffusers/pipelines/controlnet/pipeline_controlnet_inpaint.py

```diff
@@ -736,6 +739,24 @@ def check_inputs(
                     f" {negative_prompt_embeds.shape}."
                 )
 
+        if padding_mask_crop is not None:
+            if self.unet.config.in_channels != 4 and self.unet.config.in_channels != 9:
+                raise ValueError(
+                    f"The UNet should have 4 or 9 input channels for inpainting mask crop, but has"
+                    f" {self.unet.config.in_channels} input channels."
+                )
```

Comment:
```suggestion
```

we can simply remove this warning now that we accept both inpainting specific and non-specific unet